### PR TITLE
conf changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: tests/e2e/conf/prod-us-east.conf [hunk 1]
```diff
@@ -69,3 +69,10 @@ TEST_SUITE_RUN=basic
 # Testcase variables
 PLUGIN_NAME=ibm.io/ibmc-block
 PVC_ACCESS_MODE=RWO
+
+# Volume creation varibles
+OLD_REQUEST_URL=containers.bluemix.net
+NEW_REQUEST_URL=containers.bluemix.net
+OLD_REGION="region: us-east"
+NEW_REGION="region: $ARMADA_REGION"
```
Comment (akgunjal) [inline tests/e2e/conf/prod-us-east.conf:77]: You dont need the `$ARMADA_REGION` variable. Instead you can set the region as `us-east` here as this conf file is specific to us-east region.

File: tests/e2e/conf/stage.conf [hunk 2]
```diff
@@ -66,3 +66,10 @@ TEST_SUITE_RUN=basic
 # Testcase variables
 PLUGIN_NAME=ibm.io/ibmc-block
 PVC_ACCESS_MODE=RWO
+
+# Volume creation varibles
+OLD_REQUEST_URL=containers.bluemix.net
+NEW_REQUEST_URL=containers.stage1.bluemix.net
+OLD_REGION="region: us-east"
+NEW_REGION="region: $ARMADA_REGION"
```
Comment (akgunjal) [inline tests/e2e/conf/stage.conf:74]: You dont need the `$ARMADA_REGION` variable. Instead you can set the region as `us-south` here as this conf file is specific to stage which is in us-south region.

**Review by akgunjal — CHANGES_REQUESTED** [review]
I also dont see the unused variables removed from conf files. So this needs to be cleaned up.

File: tests/e2e/conf/prod-us-east.conf (hunk 1 above)
Comment (mumutyal) [reply tests/e2e/conf/prod-us-east.conf:77]: That was old variable being used to create cluster and other operations using the same

**Review by akgunjal — APPROVED** [review]

**Pull request merged**